Controller/AuthorizeController.php

```diff
@@ -52,14 +52,14 @@ public function authorizeAction(Request $request)
         $form = $this->container->get('fos_oauth_server.authorize.form');
         $formHandler = $this->container->get('fos_oauth_server.authorize.form.handler');
 
+        $scope = $request->query->get('scope', $request->request->get($form->getName().'[scope]', null, true));
```

Comment:
The way [`AuthorizeController::processSuccess()`](https://github.com/FriendsOfSymfony/FOSOAuthServerBundle/blob/master/Controller/AuthorizeController.php#L100-L102) works is by putting all fields userneath the forms namespace in `$request->query`. The OAuth library used requires `scope` to have that parameter name, so making it dynamic seems illogical.

However, I could use `$form->get('scope')->getData()` instead of reading from the request directly. Is that more in line with your expectations? The field name actually is still hardcoded that way.